backend/app/models/question.rb

```diff
@@ -7,7 +7,7 @@ class Question < ApplicationRecord
     def is_correct?(user_answer)
 
         if is_integer?(answer)
-            NumbersInWords.in_numbers(user_answer) == answer.to_i           
+            NumbersInWords.in_numbers(user_answer) == answer.to_i
```

Comment:
Could you tidy up this file and follow the spaces convention rule? Thanks @ninariccimarie